Sources/FrontEnd/Parse/Parser.swift

```diff
@@ -1066,7 +1066,8 @@ public enum Parser {
         site: state.range(from: prologue.startIndex)))
   }
 
-  /// Returns the specified access modifier of the provided `prologue`, or synthesizes an implicit one
+  /// Returns the specified access modifier of the provided `prologue`, or synthesizes an implicit
+  /// one.
```

Comment:
```suggestion
  /// Returns the access modifier in `prologue` if in contains one, or synthesizes an implicit one.
```
In an attempt to get rid of the newline.